src/pages/Dashboard/Portfolio.tsx

```diff
@@ -32,6 +33,10 @@ export const Portfolio = () => {
   const loading = useSelector(selectPortfolioLoading)
   const isLoaded = !loading
 
+  const isRainbowChartsEnabled = useFeatureFlag('RainbowCharts')
+  const [isRainbowChart, setIsRainbowChart] = useState(isRainbowChartsEnabled)
```

Comment:
```suggestion
  const [isRainbowChart, setIsRainbowChart] = useState(false)
```

when you're finished devving